Dockerfile

```diff
@@ -23,7 +23,7 @@ LABEL org.opencontainers.image.source="https://github.com/docker-mailserver/dock
 
 ENV ENABLE_POSTGREY=0
 ENV FETCHMAIL_POLL=300
-ENV ONE_DIR=0
+ENV ONE_DIR=1
```

Comment:
Does anyone knows, whats the purpose of having this ENV here?

Reply:
I think this variable is read in some scripts / needed somewhere even before it is read in `start-mailserver.sh`.

Reply:
> somewhere even before it is read in start-mailserver.sh

Startup: `dumb-init --> supervisor --> start-mailserver.sh`

I couldn't find / think of a place that comes before `start-mailserver.sh`. The same goes as well for the other ENVs.
Maybe in a separate PR we can give it a try (removing the ENVs) and see if tests pass 😃